source/slang/diagnostic-defs.h

```diff
@@ -447,6 +447,7 @@ DIAGNOSTIC(51092, Error, stageDoesntHaveInputWorld, "'$0' doesn't appear to have
 
 DIAGNOSTIC(52000, Error, multiLevelBreakUnsupported, "control flow appears to require multi-level `break`, which Slang does not yet support");
 
+DIAGNOSTIC(52001, Warning, dxilNotFound, "dxil.dll not found, so 'dxc' output cannot be signed! Shader code will not be runnable in non-development environments.");
```

Comment:
Minor nit, but I suppose we could try to phrase this so that it doesn't assume a `.dll` suffix for shared libraries.